GetUserInfo/GetUserInfo/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GetUserInfo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            UserInfo UI = Questions();
+            Console.WriteLine("Do you have any pets?");
+            string response = Console.ReadLine();
+            List<string> petNames = new List<string>();
```

Comment:
Could the pet names be added as another property of the UserInfo class perhaps?  This `petNames` variable seems disconnected from the other aspects of the user that are properties of UserInfo